packages/drivers/odsp-driver/src/odspDriverUrlResolverForShareLink.ts

```diff
@@ -121,9 +120,9 @@ export class OdspDriverUrlResolverForShareLink implements IUrlResolver {
 
     private toInstrumentedTokenFetcher(
         logger: ITelemetryLogger,
-        tokenFetcher: TokenFetcher<SharingLinkTokenFetchOptions>,
-    ): TokenFetcher<SharingLinkTokenFetchOptions> {
-        return async (options: SharingLinkTokenFetchOptions) => {
+        tokenFetcher: TokenFetcher<OdspResourceTokenFetchOptions>,
+    ): TokenFetcher<OdspResourceTokenFetchOptions> {
```

Comment:
Since we no longer decide on the resource of the token using `type` from `SharingLinkTokenFetchOptions`, will the TokenFetcher supplied here always return the OneDrive token? We no longer need it to supply a Graph token?

Reply:
Correct, this was primary motivation for the change: reduce the number of tokens that we need to produce share link.

Reply:
Yes, that's correct. This is primary motivator for this change: to eliminate dependency on Graph token and simplify things.